Makefile

```diff
@@ -63,3 +63,6 @@ validate-wgsl: $(SNAPSHOTS_OUT)/*.wgsl
 		echo "Validating" $${file#"$(SNAPSHOTS_OUT)/"};	\
 		cargo run --features wgsl-in $${file}; \
 	done
+
+
+validate-hlsl: $(SNAPSHOTS_OUT)/*.hlsl
```

Comment:
I couldn't get any for loop variant for Powershell or Command line language (for Windows) working here. I also tried the for loop that is used for the other make functions, but none of them work on my Windows system.

When we solve this, I think the PR is 99% done.

Reply:
This is indeed tough, unexpectedly!
I found this to work:
```bash
validate-hlsl: $(SNAPSHOTS_OUT)/*.hlsl
	@FOR %%f IN ($^) DO dxc %%f -T ps_6_0
```